<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PradoEditor</class>
 <widget class="QMainWindow" name="PradoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>915</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed"/>
  </property>
  <property name="windowTitle">
   <string>Prado Museum App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">

      <!-- Home Page -->
      <widget class="QWidget" name="pageHome">
       <layout class="QVBoxLayout" name="homeLayout">
        <item>
         <widget class="QLabel" name="labelPradoTitle">
          <property name="text">
           <string>Prado Museum</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="galleryListView"/>
        </item>
        <item>
         <widget class="QLabel" name="labelIntro">
          <property name="text">
           <string>Welcome to the Prado Museum App!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="homeButtonsLayout">
          <item>
           <widget class="QPushButton" name="btnExpand">
            <property name="text">
             <string>Expand Artwork</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFavorites">
            <property name="text">
             <string>My Favorites</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <!-- Artwork Details Page -->
      <widget class="QWidget" name="pageDetails">
       <layout class="QVBoxLayout" name="detailsLayout">
        <item>
         <widget class="QLabel" name="labelArtworkTitle">
          <property name="text">
           <string>Artwork Title</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelArtworkImage">
          <property name="text">
           <string>[Artwork Image Placeholder]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelArtworkDescription">
          <property name="text">
           <string>Artwork description goes here.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="detailsButtonsLayout">
          <item>
           <widget class="QPushButton" name="btnEditImage">
            <property name="text">
             <string>Edit Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFavorite">
            <property name="text">
             <string>Save to Favorites</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <!-- Filter/Edit Page -->
      <widget class="QWidget" name="pageFilter">
       <layout class="QVBoxLayout" name="filterLayout">
        <item>
         <widget class="QLabel" name="labelPreview">
          <property name="text">
           <string>[Filter Preview Image]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="filterButtons">
          <item>
           <widget class="QComboBox" name="filterDropdown"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnApplyFilter">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveFiltered">
          <property name="text">
           <string>Save Edited</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="actionButtons">
          <item>
           <widget class="QPushButton" name="btnReset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <!-- Favorites Page -->
      <widget class="QWidget" name="pageFavorites">
       <layout class="QVBoxLayout" name="favoritesLayout">
        <item>
         <widget class="QListView" name="favoritesListView"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBackFromFavorites">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
